apps/docs/src/guide/transactions/snippets/assemble-tx/change-output-collision.ts

```diff
@@ -0,0 +1,33 @@
+import type { AccountCoinQuantity } from 'fuels';
+import { Provider, ScriptTransactionRequest, Wallet } from 'fuels';
+
+import {
+  LOCAL_NETWORK_URL,
+  WALLET_PVT_KEY,
+  WALLET_PVT_KEY_2,
+} from '../../../../env';
+
+const provider = new Provider(LOCAL_NETWORK_URL);
+const baseAssetId = await provider.getBaseAssetId();
+const accountA = Wallet.fromPrivateKey(WALLET_PVT_KEY, provider);
+const accountB = Wallet.fromPrivateKey(WALLET_PVT_KEY_2, provider);
+
+// #region change-output-collision
+const request = new ScriptTransactionRequest();
+
+// Add a change output to the transaction request
+request.addChangeOutput(accountB.address, baseAssetId);
+
+const accountCoinQuantities: AccountCoinQuantity[] = [
```

Comment:
Should the snippet use the `accountCoinQuantities` constant to trigger the undesired behavior? The relationship between the `request` instance and the `accountCoinQuantities` constant may not be obvious.

Reply:
I have decided to remove the error section:

https://github.com/FuelLabs/fuels-ts/pull/3747/commits/801122619bafde519e4bd924c06c7003433b059a